docs/technical-reference/pipeline.md

```diff
@@ -116,6 +116,28 @@ Then, we will load the dataset and call the `generate` method of the pipeline. F
 
 With a few lines of code, we have easily generated a dataset with 2 generations per input, using 4 different `LLM`s, and labelled the generations using GPT-4. You can check the full code [here](https://github.com/argilla-io/distilabel/blob/main/examples/pipeline-preference-dataset-llmpool.py).
 
+### Dataset checkpoints
+
+With long pipelines, it may be useful to review the dataset during the process, just to keep track and stop the process, or have it saved in case something fails before obtaining the final dataset. We can use the `checkpoint_strategy` in `Pipeline.generate` method for this end:
```

Comment:
I don't understand the sentence "just to keep track and stop the process".

Reply:
It should be an or. But anyway I see that it doesn't make much sense. Removing that altogether